stories/Chessboard.stories.tsx

```diff
@@ -451,40 +437,42 @@ export const StyledBoard = () => {
 ///////////////////////////////////
 ////////// Custom Square ///////////
 ///////////////////////////////////
-const CustomSquareRenderer = forwardRef<HTMLDivElement, CustomSquareProps>((props, ref) => {
-  const { children, square, squareColor, style } = props;
+const CustomSquareRenderer = forwardRef<HTMLDivElement, CustomSquareProps>(
```

Comment:
My pretttier overrided this and other changes. Can we add some `prettierrc` file in separate PR?